Makefile

```diff
@@ -42,7 +42,7 @@ deflake:
 battletest: strongertests
 	go tool cover -html coverage.out -o coverage.html
 
-verify: codegen ## Verify code. Includes dependencies, linting, formatting, etc
+verify: ## Verify code. Includes dependencies, linting, formatting, etc
```

Comment:
We need to run codegen on `make ci` to ensure we catch any uncommitted changes.

Reply:
I see the impl now. We probably need to split this to avoid losing codegen checks for static code.